scripts/list_boards.py

```diff
@@ -95,18 +153,105 @@ def parse_args():
 def add_args(parser):
     # Remember to update west-completion.bash if you add or remove
```

Comment:
needs update (and I guess we have other completion scripts too now...)

Reply:
fixed